projects/components/src/search-box/search-box.component.ts

```diff
@@ -7,18 +7,20 @@ import { IconSize } from '../icon/icon-size';
   styleUrls: ['./search-box.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="htc-search-box">
-      <htc-icon icon="${IconType.Search}" size="${IconSize.Large}" class="icon" (click)="onSubmit()"></htc-icon>
+    <div class="htc-search-box" [class.focused]="this.isFocused">
+      <htc-icon icon="${IconType.Search}" size="${IconSize.Medium}" class="icon" (click)="onSubmit()"></htc-icon>
       <input
+        class="input"
         type="text"
         [placeholder]="placeholder"
-        class="input"
         [(ngModel)]="value"
-        (input)="onValueChange()"
-        (keyup.enter)="onSubmit()"
+        (input)="this.onValueChange()"
+        (keyup.enter)="this.onSubmit()"
+        (focus)="this.isFocused = true"
```

Comment:
How does `isFocused` gets set to false once the input is in focus?

Reply:
Line 20 should do it.

Reply:
oh. I missed that